### DISCUSSION TITLE
Doc review "first time user flow" perspective

### DISCUSSION THREAD

**Discussion by amandamartin-dev** [EddieHubCommunity/LinkFree#2511] [General]
@eddiejaoude Wasn't sure the best way to approach this review, but take a look and anything you would like to move forward with I am happy to create issues for. Let me know if anything is unclear!

## General notes:
If checked, issue created

- [x]  Accessibility question - the links in Documentation section don't both have underline and hover color, should this be updated?
- [x] - Nice to have "copy code" button for all code blocks
- [x] - At the end of the quickstart, I don't know where to go next to add other sections so I am confused and start clicking around to find these directions. Perhaps a "this will get you started with XYZ. To build out other sections such as Events, head to the docs (or wherever)"
- [x] - Removed - this was already taken care of by a link to saras profile in the doc that i may have missed
- [x] - Nice to have: Each image referring to an action the user needs to take should clearly highlight in the image the button, link, etc.  This would be nice as the images are mostly repo screenshots so if a person was newer to github they may have to work a little to find the button referenced in each step.  However, i call this "nice to have" because replacing/editing all the images may not be worth the time.

## During the flow notes:

- [x]  Step 2: clicking on the linkfree repo link should open in a new tab so that your directions are preserved in your current window (edit: this is in the other issue with links with notes on how to do this in an accessible way)
- [x] - Step 5: I would mention that after forking the user will want to turn off all the jobs running and link to show where: https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow
- [x] - Step 7: I would mention if this is case sensitive (github user name)
- [x] - Step 8: link to and mention that there are more fields you can include bc if I check out eddies profile as the example I’m left wondering about the tags and other features.  I understand not overwhelming a new user in a Quickstart, but perhaps a - here’s where to find the full set of fields in the docs
- [x] - Step 9 or 8: note that the curly braces should close the entire file otherwise the flow appears that you should append each block as you go which will result in errors on the json checks
- [x] - Step 9: link to the react icons resource https://react-icons.github.io/react-icons/icons?name=fa

**Comment by eddiejaoude** [comment]
WOW Amanda this is amazing, thank you ❤️ 

Please raise them all, they all sound like great improvements 🔥 - I can help raise also if you need, just let me know

**Comment by amandamartin-dev** [comment]
Happy to organize it all into issues. Might take me a day or so to get back to it. On the road tomm

**Reply by eddiejaoude** [reply]
Awesome! No problem. If you need anything let me know. Safe journey

**Comment by eddiejaoude** [comment]
> Step 9: link to the react icons resource https://react-icons.github.io/react-icons/icons?name=fa

We now have a "Search available Icons" page within the app 🎉  https://linkfree.eddiehub.io/icons

**Comment by amandamartin-dev** [comment]
@eddiejaoude sorry for the delay, all these issues have been created now and this discussion can be closed

**Reply by eddiejaoude** [reply]
no delay Amanda! Thank you for raising those issues ❤️
